mail_tracking/models/mail_message.py

```diff
@@ -58,20 +63,42 @@ def tracking_status(self):
             for partner in partners:
                 # If there is partners not included, then status is 'unknown'
                 partner_trackings.append((
-                    'unknown', False, partner.name, partner.id))
+                    'unknown', False, partner.name, partner.id, partner.email))
             res[message.id] = partner_trackings
         return res
 
+    @api.model
```

Comment:
According to your code, this fits better:

```suggestion
    @api.multi
```